File: UI/src/app/domain/equipment-base.ts
/** Модель оборудования */
export class EquipmentBase {
    /** Идентификатор обродования в справочнике */
    id:number;
    /** Наименование */
    name:string;

    constructor(id:number, name:string){
        this.id=id;
        this.name=name;
    }
}

File: Api/Reception.Api/Application/Reception.Handlers/Equipment/EquipmentBase.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Reception.Handlers.Equipment
{
    /// <summary>
    /// Обрудование.
    /// </summary>
    public class EquipmentBase
    {
        /// <summary> Идентификатор обродования в справочнике </summary>
        public int Id { get; set; }
        /// <summary> Наименование </summary>
        public string Name { get; set; }
    }
}
